nvml/README.md

```diff
@@ -0,0 +1,79 @@
+# Agent Check: Nvidia NVML
+
+## Overview
+
+This check monitors [NVIDIA Management Library (NVML)][1] exposed metrics through the Datadog Agent and can correlate them with the [exposed Kubernetes devices][12].
+
+## Setup
+
+The NVML check is not included in the [Datadog Agent][2] package, so you need to install it.
+
+### Installation
+
+For Agent v7.21+ / v6.21+, follow the instructions below to install the NVML check on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.
+
+1. Run the following command to install the Agent integration:
+
+   ```shell
+   datadog-agent integration install -t datadog-nvml==<INTEGRATION_VERSION>
+   # You may also need to install dependencies since those aren't packaged into the wheel
+   sudo -u dd-agent -H /opt/datadog-agent/embedded/bin/pip3 install grpcio pynvml
+   ```
+
+2. Configure your integration similar to core [integrations][4].
+
+If you are using Docker, there is an [example Dockerfile][15] in the NVML repository.
+
+   ```shell
+   docker build -t dd-agent-nvml .
+   ```
+
+If you're using Docker and Kubernetes, you need to expose the environment variables `NVIDIA_VISIBLE_DEVICES` and `NVIDIA_DRIVER_CAPABILITIES`. See the included Dockerfile for an example.
+
+To correlate reserved Kubernetes NVIDIA devices with the Kubernetes pod using the device, mount the Unix domain socket `/var/lib/kubelet/pod-resources/kubelet.sock` into your Agent's configuration. More information about this socket is on the [Kubernetes website][12]. **Note**: This device is in beta support for version 1.15.
+
+### Configuration
+
+1. Edit the `nvml.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your NVML performance data. See the [sample nvml.d/conf.yaml][7] for all available configuration options.
+
+2. [Restart the Agent][8].
+
+### Validation
+
+[Run the Agent's status subcommand][9] and look for `nvml` under the Checks section.
```

Comment:
Could you update these instructions since now the check will be shipped with the agent?